### DISCUSSION TITLE
F-droid 发布

### DISCUSSION THREAD

**Discussion by zoenglinghou** [LeetaoGoooo/RSSAid#17] [Ideas]
请问这个程序能通过 F-droid 分发吗？粗略看了一下 Features 感觉 RSSAid 是一个自由软件，完全符合在 F-droid 上分发的标准。

通过 F-droid 分发的话，虽然发布时间会有延迟，但是不用考虑自动更新的问题，也能增加受众面积。

**Comment by LeetaoGoooo** [comment]
> 请问这个程序能通过 F-droid 分发吗？粗略看了一下 Features 感觉 RSSAid 是一个自由软件，完全符合在 F-droid 上分发的标准。
> 
> 通过 F-droid 分发的话，虽然发布时间会有延迟，但是不用考虑自动更新的问题，也能增加受众面积。

之前不知道 F-droid ，简单看了一下应该可以～～后续试着在这个平台发布

**Comment by stonega** [comment]
@zoenglinghou @LeetaoGoooo 发布到 fdroid 我以前有做过，这个 assign 给我吧

**Reply by libook** [reply]
请问这个在fdroid上发布了嘛？目前搜不到

**Comment by LeetaoGoooo** [comment]
> @zoenglinghou @LeetaoGoooo 发布到 fdroid 我以前有做过，这个 assign 给我吧

好的～辛苦了，发布正式版本就好了～～pre release 就不用了😂

**Comment by stonega** [comment]
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9762
Sorry 鸽了这么久， 已经向Fdroid提交了PR，等待合并中。

**Reply by LeetaoGoooo** [reply]
> https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9762
> Sorry 鸽了这么久， 已经向Fdroid提交了PR，等待合并中。

辛苦了～
